engine/src/exec-store.cpp

```diff
@@ -155,8 +155,9 @@ void MCStoreGetPurchaseError(MCExecContext& ctxt, int p_id, MCStringRef& r_error
     
     t_success = MCPurchaseFindById(p_id, t_purchase);
 	
+    // PM-2015-01-19: [[ Bug 14401 ]] Fixed mismerge issue that caused mobileStorePurchaseError to return empty 
```

Comment:
Sorry for the confusion, "mismerge" may not be the correct word. I mean that the original code which worked on 6.7, was not refactored properly so as to work on 7.0.